src/graphql/home/featuredBuyNowListings.graphql

```diff
@@ -0,0 +1,5 @@
+query featuredBuyNowListings($limit: Int!) {
+  nfts(limit: $limit, offset:0, auctionHouses: ["9SvsTjqk3YoicaYnC4VW1f8QAN9ku7QCCk6AyfUdzc9t"], listed: true) {
+    address
```

Comment:
Does this just need the address and nothing more?

Reply:
I think so. The API should be minimal, because the server should primarily determine what's a "Hot" NFT. For now this is just returning any NFT in the Holaplex marketplace so that it is limited to Buy Now listings. That said, the results dont look great